variables.tf

```diff
@@ -195,3 +195,7 @@ variable "datapipeline_config" {
     timeout       = "60 Minutes"
   }
 }
+
+variable "use_efs_ip_address" {
+  default = false
```

Comment:
Use string values. They'll be interpolated as booleans in TF, if boolean is expected.

> Although it appears Terraform supports boolean types, they are instead silently converted to string types. The implications of this are subtle and should be completely understood if you plan on using boolean values.

https://www.terraform.io/docs/configuration/variables.html